saskatoon/harvest/locale/fr/LC_MESSAGES/django.po

```diff
@@ -57,695 +57,750 @@ msgstr "Validation en attente"
 msgid "Notes from the pick leader."
-msgstr ""
+msgstr "Notes du Chef (de la Cheffe) de cueillette"
 
 #: harvest/models.py:581 harvest/models.py:633
 msgid "Harvest"
-msgstr ""
+msgstr "Récolte"
 
 #: harvest/models.py:587
 msgid "Created on"
-msgstr ""
+msgstr "Créé le"
 
 #: harvest/models.py:592
 msgid "Accepted on"
-msgstr ""
+msgstr "Accepté le"
 
 #: harvest/models.py:598
 msgid "Accepted"
-msgstr ""
+msgstr "Accepté"
 
 #: harvest/models.py:605
 msgid "Showed up"
-msgstr ""
+msgstr "A participé"
 
 #: harvest/models.py:612
 msgid "Canceled"
-msgstr ""
+msgstr "Annulée"
 
 #: harvest/models.py:617
 msgid "request for participation"
-msgstr ""
+msgstr "demande de participation"
 
 #: harvest/models.py:618
 msgid "requests for participation"
-msgstr ""
+msgstr "demandes de participation"
 
 #: harvest/models.py:644
 msgid "Weight (lb)"
-msgstr ""
+msgstr "Poids (en livres)"
 
 #: harvest/models.py:652
 msgid "Recipient"
-msgstr ""
+msgstr "Bénéficiaire"
 
 #: harvest/models.py:657
 msgid "harvest yield"
-msgstr ""
+msgstr "rendement de cueillette"
 
 #: harvest/models.py:658
 msgid "harvest yields"
-msgstr ""
+msgstr "rendements de cueillettes"
 
 #: harvest/models.py:673
 msgid "Description"
-msgstr ""
+msgstr "Description"
 
 #: harvest/models.py:695
 msgid "Shared"
-msgstr ""
+msgstr "Partagé"
 
 #: harvest/models.py:696
 msgid "Can be used in harvests outside of property"
-msgstr ""
+msgstr "Peut être utilisé pour des cueillettes ailleurs"
 
 #: harvest/models.py:701 harvest/models.py:702
 msgid "equipment"
-msgstr ""
+msgstr "équipement"
 
 #: harvest/models.py:710
 msgid "Content"
-msgstr ""
+msgstr "Contenu"
 
 #: harvest/models.py:715
 msgid "Created date"
-msgstr ""
+msgstr "Date de création"
 
 #: harvest/models.py:721
 msgid "Author"
-msgstr ""
+msgstr "Auteur"
 
 #: harvest/models.py:734
 msgid "comment"
-msgstr ""
+msgstr "commentaire"
 
 #: harvest/models.py:735
 msgid "comments"
-msgstr ""
+msgstr "commentaires"
 
 #: harvest/views.py:25
 msgid "Equipment created successfully!"
-msgstr ""
+msgstr "Création d'équipement réussie !"
 
 #: harvest/views.py:29
 msgid "Add new equipment"
-msgstr ""
+msgstr "Ajouter un nouvel équipement"
 
 #: harvest/views.py:38
 msgid "Equipment updated successfully!"
-msgstr ""
+msgstr "Mise à jour d'équipement réussie !"
 
 #: harvest/views.py:43
 msgid "Edit equipment"
-msgstr ""
+msgstr "Modifier l'équipement"
 
 #: harvest/views.py:56
 msgid "Property created successfully!"
-msgstr ""
+msgstr "Création de propriété réussie !"
 
 #: harvest/views.py:60
 msgid "Add a new property"
-msgstr ""
+msgstr "Ajouter une nouvelle propriété"
 
 #: harvest/views.py:70
 msgid ""
 "Thanks for adding your property! In case you authorized a harvest for this "
-"season, please read the <a href='https://core.lesfruitsdefendus.org/s/"
-"bnKoECqGHAbXQqm'>Tree Owner Welcome Notice</a>."
+"season, please read the <a "
+"href='https://core.lesfruitsdefendus.org/s/bnKoECqGHAbXQqm'>Tree Owner "
```

Comment:
is this link up to date? would probably be best to serve a static PDF just like we do with Equipment points and Volunteer Waiver